Tests/timestamp_replacer.py

```diff
@@ -213,8 +213,14 @@ def clean_json_body(self, req: HTTPRequest) -> None:
                 try:
                     content = OrderedDict(literal_eval(content))
                     self.modify_json_body(req, content)
+                    return
                 except Exception:
                     logging.exception(f'failed to run literal_eval on content {content}')
+                try:
+                    content = json.loads(content, object_pairs_hook=OrderedDict)
+                    self.modify_json_body(req, content)
+                except Exception:
+                    logging.exception(f'failed to run json.loads on content {content}')
```

Comment:
If you nest this `try`-`except` block under the first exception handler, then there is no need for the `return` statement.

Reply:
Correct but that's weird.
I personally don't like nested code
let me know how important it is for you, if it is I'll change it